components/Chart.tsx

```diff
@@ -258,19 +258,19 @@ const Chart = (props: Props) => {
     .value();
 
   // Ensure totalsByPatternClass has an entry for **every** pattern class, insert one if it doesn't
-  if (totalsByPatternClass.length !== patternClasses?.length) {
-    patternClasses?.forEach(globalPatternClass => {
-      if (!_.find(totalsByPatternClass, ['name', globalPatternClass.name])) {
-        totalsByPatternClass.push({
-          terms: [],
-          meta: globalPatternClass,
-          name: globalPatternClass.name,
-          avgRights: 0,
-          avgObligations: 0,
-        });
-      }
-    });
-  }
+  // if (totalsByPatternClass.length !== patternClasses?.length) {
+  //   patternClasses?.forEach(globalPatternClass => {
+  //     if (!_.find(totalsByPatternClass, ['name', globalPatternClass.name])) {
+  //       totalsByPatternClass.push({
+  //         terms: [],
+  //         meta: globalPatternClass,
+  //         name: globalPatternClass.name,
+  //         avgRights: 0,
+  //         avgObligations: 0,
+  //       });
+  //     }
+  //   });
+  // }
```

Comment:
just comment-out this bit of logic for now as it might make sense to wrap in an optional boolean chart prop later (eg `includeEmptyClasses` or something??)